localization/english/load_tips/load_tips_l_english.yml

```diff
@@ -0,0 +1,114 @@
+ LOADING_TIP_59:0 "“We may dress differently... or seem odd... but we're really just like everyone else. We value the same things you do... money, happiness, family... Yes, we're a family. We're a family that, like every other rube family out there, takes care of our own. We just do it our own way.”\n #bold —Silas to Cedrick Fallrook, shortly before locking him inside a coffin and burying Cedrick alive.#!"
+ LOADING_TIP_60:0 "#T Did you know?#!\n\nThe Monkey King is a hozen who was a companion of Shaohao, the last emperor of Pandaria."
+ LOADING_TIP_61:0 "#T Did you know?#!\n\nCho'gall was once claimed to be one of the first ogre-magi, or indeed the first."
+ LOADING_TIP_62:0 "“I have watched the other races... I have seen their squabbling, their ruthlessness. Their wars do nothing but scar the land, and drive the wild things to extinction. No, they cannot be trusted. Only beasts are above deceit.”\n #bold —Rexxar#!"
+ LOADING_TIP_63:0 "#T Did you know?#!\n\nRexxar claims to be, in his own words, the "last son of the Mok'Nathal""
+ LOADING_TIP_64:0 "#T Did you know?#!\n\nThe Nathrezim, also known as the Dread Lords, are most commonly associated with the Burning Legion. However, there are a few who serve the Army of the Light."
+ LOADING_TIP_65:0 "“When defeat is impossible, victory becomes inevitable.”\n #bold —Blademaster Telaamon#!" 
+ LOADING_TIP_66:0 "#T Did you know?#!\n\nYou can find very interesting people in the human town of Goldshire."
+ LOADING_TIP_67:0 "“She had perfected the art of gentle persuasion, and her love moved the world.”\n #bold —Varian Wrynn about his late wife Tiffin#!"
+ LOADING_TIP_68:0 "#T Did you know?#!\n\nMillhouse Manastorm was "accidentally" imprisioned in the Arcatraz by the Naaru."
+ LOADING_TIP_69:0 "#T Did you know?#!\n\nA trade prince of Bilgewater Cartel, Gazlowe, once invited Harleen Chirpsnide over and cooked dinner to impress her. She says it smelled bad."
+ LOADING_TIP_70:0 "“Oh my - as I suspected. Your ancestry dates back to the dawn of the world. Great were the ancient empires of the trolls. I see a spark in your eyes, a powerful will - you wish to be great again, yes?”\n #bold —Lorewalker Cho, speaking about the Trolls#!"
+ LOADING_TIP_71:0 "#T Did you know?#!\n\nSand trolls are a subgroup of jungle trolls that dwell in the Tanaris desert. The subgroup arose by adapting to the beating of the hot Tanaris sun after being separated from their Eastern Kingdoms brethren in the Great Sundering."
+ LOADING_TIP_72:0 "#T Did you know?#!\n\nBlood trolls are a troll subgroup who inhabit the swamps of Nazmir on Zandalar and worship the blood god G'huun. They use forbidden blood magic, which corrupts the creatures around them and turns them into blood beasts. The blood trolls originate from Zandalari trolls who long ago clung to the use of blood magic."
+ LOADING_TIP_73:0 "#T Did you know?#!\n\nIn the main building of Mistfall Village, a bush chicken is playing a troll guitar in hopes of spare change."
```

Comment:
LOADING_TIP_73:0 "#T Did you know?#!\n\nIn the main building of Mistfall Village, a bush chicken is playing a troll guitar in the hopes of spare change."

Reply:
thats u